doc(release) Release notes oss 24.10 (#4429)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

## Centreon Web

### 24.10.8

> Release date: `June 19, 2025`

Reviewer: ```suggestion
Release date: `June 20, 2025`
```

<details open>
  <summary>Enhancements</summary>

> - [Agent configuration] Added a "No TLS" connection mode for CMA (limited) & Telegraf.

Reviewer: ```suggestion
- [Agent configuration] Added a "No TLS" connection mode for CMA & Telegraf. For CMA, this mode is only active within a limited timeframe.
```

> - [Agent configuration] Sub-directories are now allowed in certificate configuration.

Reviewer: ```suggestion
- [Agent configuration] Sub-directories are now allowed in certificate configuration fields.
```

</details>

<details>
  <summary>Bug fixes</summary>

> - [Configuration] Command is now mandatory when Add/Edit a service.

Reviewer: ```suggestion
[Configuration] Cases where no check command is defined when creating or editing a service are now detected before saving.
```

> - [Configuration] Fixed an issue when adding/editing service when command's template is not the direct parent.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where adding/editing a service failed when a command was configured on a template that was not the direct parent of the service.
```

> [Configuration] Cases where no check command is defined when creating or editing a service are now detected before saving.

Reviewer: ```suggestion
- [Configuration] Cases where no check command is defined when creating or editing a service are now detected before saving.
```

> - [Configuration] Fixed an issue which blocked Time period entries with more than 200 characters in UI.

Reviewer: ```suggestion
- [Configuration] Time period names can now have more than 200 characters.
```

- [Custom Views] The "Battery Status" information is now available in the Gridmap widget.

> - [Dashboards][Public Playlists] Fixed issue with Metrics Graph displaying too many curves.

Reviewer: ```suggestion
- [Dashboards][Public Playlists] Fixed an issue where Metrics Graphs were displaying too many curves.
```

> - [Graph] Fixed issue preventing "fill" and "invert" options from being considered in the graphs from Resource Status and Dashboards.

Reviewer: ```suggestion
- [Graph] Fixed an issue preventing "fill" and "invert" options from being considered in the graphs in Resource Status and Dashboards.
```

> - [Install] Fixed an upgrade fail linked to duplicate entry in topology table.

Reviewer: ```suggestion
- [Install] Upgrades to version 24.10 no longer fail (a duplicate entry in the topology table has been removed).
```

> - [LDAP] Fixed a validation issue on custom filter for users and groups.

Reviewer: ```suggestion
- [LDAP] Fixed the syntax that validates custom filters for users and groups.
```

> - [Purge] Fixed an issue on data purge cron with partitioned tables.

Reviewer: ```suggestion
- [Cron] Fixed an issue where some partitioned tables were not deleted from the database.
```

> - [Remote Server] Fixed issue preventing Business Activities from being displayed on remote servers.

Reviewer: ```suggestion
- [Remote Server] Fixed an issue preventing Business Activities from being displayed on remote servers.
```

> - [Resource Status] Fixed an issue where timeline period filter was nos properly applied.

Reviewer: ```suggestion
- [Resource Status] Fixed an issue with timeline period filter not being properly applied.
```

> - [Resource Status] Fixed an issue when searching for an item containing a slash "/".

Reviewer: ```suggestion
- [Resource Status] Searching for an item containing a slash character is now handled correctly.
```

> - [Resource Status] Fixed dual display of the calculation type tile in the "Details" panel of a resource.

Reviewer: ```suggestion
- [Resource Status] In the **Details** panel of a resource, the **Calculation type** tile is no longer displayed twice.
```

> - [Status Chart widget] Displayed the correct list of resources when hovering the legend.

Reviewer: ```suggestion
- [Status Chart widget] Hovering over the legend of the donut now displays the correct list of resources.
```

</details>

<details>
  <summary>Security fixes</summary>

> - [Install] Fixed a password visibility issue on /etc/centreon/config.d/10-database.yaml.

Reviewer: ```suggestion
- [Install] Fixed incorrect rights on a database configuration file.
```

## Centreon Open Tickets

### 24.10.3

> Release date: `June 19, 2025`

Reviewer: ```suggestion
Release date: `June 20, 2025`
```

<details>
  <summary>Bug fixes</summary>

> - [Dashboards] Fixed issue preventing non admin users from opening tickets.

Reviewer: ```suggestion
- [Dashboards] Fixed an issue preventing non-admin users from opening tickets.
```
